plugins/ci/skills/bulk-triage-regressions/SKILL.md

```
@@ -0,0 +1,149 @@
+description: Holistically analyze and triage all untriaged Component Readiness regressions for a set of components in a view (triage duty workflow)
+argument-hint: <view> [--components comp1 comp2 ...] [--auto-triage]
+example: "/ci:triage-component-regressions 5.0-main --components Installer Unknown"
+---
+
+## Name
+
+ci:triage-component-regressions
+
+## Synopsis
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)